Trustroots/trustroots
# Update homepage screenshots

Kilobyte-optimize and refresh screenshot images on the homepage.

Faster loading times, happier people, happier search engines. ✨ 

#### Proposed Changes

- Add a new Screenshot component
- Optimize images by serving Webp format for capable browsers (see [`picture`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) element)
- Update screenshots to fresh ones

Profile photo in screenshot via https://search.creativecommons.org/photos/41220d20-f311-4bd3-8329-6d2e36e808fe

**Before**

![image](https://user-images.githubusercontent.com/87168/101988953-b7737680-3ca5-11eb-9010-f6be4bb8be70.png)


**After**

<img width="1587" alt="Screenshot 2020-12-12 at 17 56 36" src="https://user-images.githubusercontent.com/87168/101988945-a165b600-3ca5-11eb-8fb2-9eb7b0f7ec65.png">


#### Testing Instructions

* Check unlogged homepage and observe screenshots are intact
* Try non-retina (2x hires), and mobile screens
* Ensure no multiple versions of the same image are loading


Resolves https://github.com/Trustroots/trustroots/issues/1367

Alternative to https://github.com/Trustroots/trustroots/pull/1465
